discoveries/fire-hydro-impacts.discoveries.mdx

```diff
@@ -0,0 +1,118 @@
+---
+featuredOn:
+  - eis
+id: "fire-hydro-impacts"
+name: Post-Fire Hydrology Impacts in Western US
+description: "High resolution ET observations reveal hydrological response in Westen US"
+media:
+  src: ::file ./fire-cover.png
+  alt: Satellite imagery (false color) of East Troublesome fire spreading to the Rockies
+  author:
+    name: Lauren Dauphin/Landsat 8
+    url: https://eoimages.gsfc.nasa.gov/images/imagerecords/147000/147452/coloradofire752_oli_2020296_lrg.jpg
+pubDate: 2020-12-01
+thematics:
+  - eis
+---
+
+<Block>
+  <Prose>
+    ## When Fires Disturb Eco-Hydrology
+
+    Fire, either natural or otherwise, is a pervasive ecological and hydrological disturbance that sparks extensive impacts to underlying vegetation, soil, and the ecosystem. Fires modulate ecosystem evapotranspiration (ET) by directly affecting the aboveground vegetation immediately after burning, thereby reducing transpirational leaf surface area.
+  </Prose>
+</Block>
+
+<Block>
+  <Prose>
+    The Western United States (WUS) is experiencing an increase of catastrophic wildfires. While fires historically constituted an important rangeland disturbance process in the semi-arid ecosystem of WUS, climate change and intensifying droughts have increased vigorous fire activities 
+  </Prose>
+  <Figure>
+    <Image
+      src={new URL("./wus_burns.png", import.meta.url).href}
+      alt="fires in western US"
+    />
+    <Caption
+      attrAuthor="NASA EIS Freshwater"
+      attrUrl="https://freshwater.eis.smce.nasa.gov/"
+    />
+  </Figure>
+</Block>
+
+<Block>
+  <Figure>
+    <Image
+      src={new URL("./et_suppression.png", import.meta.url).href}
+      alt="fires in western US"
+    />
+    <Caption
+      attrAuthor='NASA'
+      attrUrl='https://nasa.gov/'
+    >
+      Impact of fires on reduction of evapotranspiration, shown in high resolution datasets from OpenET over 2020 fires in Western US
+    </Caption>
+  </Figure>
+
+  <Prose>
+    ## Sensing Vegetation Changes from Space
+
+    We use observations of ET from high-resolution OpenET products and burned area severity from Monitoring Trends in Burn Severity (MTBS) maps to analyze the post-fire impact on hydrology and water balance.
+    Fire leads to reduction in evapotranspiration (ET) which is modeled using Open ET observations. Difference in water balance (precipitation minus ET) over fire impacted region from than over a control unimpacted region gives an estimate of ET suppression caused by fire.
+  </Prose>
+
+</Block>
+
+<Block type="full">
+  <Figure>
+    <Image
+      src={new URL("./ca-fires.jpeg", import.meta.url).href}
+      alt="fires in western US"
+    />
+    <Caption
+      attrAuthor='NASA'
+      attrUrl='https://nasa.gov/'
+    >
+      Fires in Oregon and California Grow in Size and Intensity (https://www.nasa.gov/image-feature/goddard/2018/fires-in-oregon-and-california-grow-in-size-and-intensity)
```

Comment:
It would be a better experience to not have the link spelled out, but as part of a caption sentence. Something like:

> More information about Fires in Oregon and California can be found in NASA's [Fires in Oregon and California Grow in Size and Intensity](https://www.nasa.gov/image-feature/goddard/2018/fires-in-oregon-and-california-grow-in-size-and-intensity) article.